pyproject.toml

```diff
@@ -65,3 +65,41 @@ ignore_errors = true
 
 [tool.coverage.html]
 directory = "htmlcov"
+
+[tool.mypy]
+python_version = "3.9"
```

Comment:
We actually support a larger range. Need to confirm that the type hints work as expected with other versions of Python, especially the older ones

Reply:
Removed `python_version` per conversation to rely on default behavior